platform/podjs/cypress.json

```diff
@@ -1,4 +1,5 @@
 {
     "pluginsFile": false,
-    "video": false
+    "video": false,
+    "chromeWebSecurity": false
```

Comment:
Yeah, I did not want to disable `chromeWebSecurity` but when I try to fetch the zip over https I get this error `TypeError: Failed to fetch`. I found something about it here: https://github.com/cypress-io/cypress/issues/2047, but it does not seem to be figured out. Do you guys know what is the issue?

Reply:
Well, fetching stuff from outside the repo is tricky. It adds to testing time, for instance. It might clash with sandboxing.

Reply:
Do you still need it?